modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/change/tracking/spi/reference/DDMFormInstanceTableReferenceDefinition.java

```diff
@@ -71,6 +71,9 @@ public void defineChildTableReferences(
 						DDMFormInstance.class.getName())
 				)
 			)
+		).singleColumnReference(
+			DDMFormInstanceTable.INSTANCE.structureId,
+			DDMStructureTable.INSTANCE.structureId
```

Comment:
If you take a look at the `addFormInstance` methods in [DDMFormInstanceLocalServiceImpl](https://github.com/liferay/liferay-portal/blob/master/modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/service/impl/DDMFormInstanceLocalServiceImpl.java), it never adds a DDMStructure when a new form instance is added. So DDMStructure is never a child of a form instance.

Also, the foreign key generally references a primary key in the parent table. In this case, `structureId` is the foreign key, so DDMStructure has to be the parent table.